### ISSUE TITLE
[FEATURE] Creating Grouped Time Series Structure

### ISSUE THREAD

**Issue by noahsa** [carlomazzaferro/scikit-hts#48]
**Is your feature request related to a problem? Please describe.**
It's not a problem. I am looking to create a grouped time series structure, as opposed to a hierarchical.

**Describe the solution you'd like**
I was wondering if you have any experience, suggestions, or an example would be wonderful demonstrating how to use this library to create a grouped time series structure? Or if it is possible in the current version?

**Describe alternatives you've considered**
I have looked through the documentation and have not been able to find references specific to creating grouped time series. Two attempted solutions are:

1. Create a group structured dictionary to pass to hts.hierarchy.HierarchyTree.from_nodes(). Although, I have not been able to figure out how to structure the dictionary to create the correct summing matrix.

2. Use hts.hierarchy.utils.groupify(), although similarly I was unsure how to utilize this function correctly to create a tree with a grouping structure.

**Additional context**
Please let me know if you need any additional information. This seems like a very cool library, so I would be interested in contributing if there are small things to be done to enable this and other functionality. Any guidance is appreciated.

**Comment by carlomazzaferro** [comment]
Grouped time series would operate in the same way as hierarchical time series, except that there is a single level of aggregation. For the mobility data, it would be something like:

```
hier = {
                'total': ['CH', 'SLU', 'BT', 'OTHER']
}
```

The reconciliation algorithms still apply, so you can still use the same methodologies included in the library. See https://otexts.com/fpp2/gts.html for more background

**Comment by noahsa** [comment]
Thank you for the quick response! That's great to hear that the reconciliation algorithms will still work in the library too, since that's the next step for us.  

I am still a little fuzzy though on  how to create the grouped representation from your example. If we looked at the example at the top of the FPP page you linked, how would you represent the groups using the hier dictionary?

Attributes 1 - (A,B)
Attributes 2 - (X,Y)

Where we forecast Attributes 1, Attributes 2, and Attributes 1 * Attributes 2, and total. With Attributes 1 and 2 forecasts still mapping up to total like in the diagram at the top of the FPP page?

My thought was it would be something like this:

`hier = {'total': ['A', 'B', 'X', 'Y'],
               'A': ['AX', 'AY'],
              'B': ['BX', 'BY'],
              'X': ['AX', 'BX'],
              'Y': ['AY', 'BY']}`

Although, this is clearly flawed since both Attributes 1 and 2, add up to total individually as opposed to all together. Thanks again for your help!

**Comment by noahsa** [comment]
Resolved in #51.
